public/content/developers/docs/intro-to-ethereum/index.md

```diff
@@ -44,7 +44,7 @@ ETH is also used to provide crypto-economic security to the network in three mai
 
 ## What are smart contracts? {#what-are-smart-contracts}
 
-In practice, participants don't write new code every time they want to request a computation on the EVM. Rather, application developers upload programs (reusable snippets of code) into EVM state, and users make requests to execute these code snippets with varying parameters. We call the programs uploaded to and executed by the network smart contracts.
+In practice, participants don't write new code every time they want to request a computation on the EVM. Rather, application developers upload programs (reusable snippets of code) into EVM state, and users make requests to execute these code snippets with varying parameters. We call the programs uploaded to and executed by the network as "smart contracts".
```

Comment:
Hey @vibhorkashyap, thanks for the suggestion! I could get behind the quotation marks since we're stating "We _call_ [them]" but the "as" kinda breaks that.. would remove "as"

```suggestion
In practice, participants don't write new code every time they want to request a computation on the EVM. Rather, application developers upload programs (reusable snippets of code) into EVM state, and users make requests to execute these code snippets with varying parameters. We call the programs uploaded to and executed by the network "smart contracts".
```